private/scripts/sage-theme-install.sh

```diff
@@ -182,30 +181,30 @@ function install_sage() {
   # Check if the directory $sagedir is empty. If it's not, bail.
   echo -e "Checking if ${sagedir} is exists and if it's empty.\n"
 
-  if [ "$(ls -A $sagedir)" ]; then
+  if [ "$(ls -A "$sagedir")" ]; then
     echo -e "${red}Directory not empty!${normal}\n Trying to install into ${sagedir}. Exiting."
     exit 1;
   fi
 
   echo -e "${yellow}Installing Sage.${normal}"
   # Create the new Sage theme
-  composer create-project roots/sage $sagedir
+  composer create-project roots/sage "$sagedir"
 
   # Require Roots/acorn
-  composer require roots/acorn --working-dir=$sagedir
+  composer require roots/acorn --working-dir="$sagedir"
 
   # Install all the Sage dependencies
-  composer install --no-dev --prefer-dist --working-dir=$sagedir
+  composer install --no-dev --prefer-dist --working-dir="$sagedir"
 
   # NPM the things
-  npm install --prefix $sagedir
-  npm run build --prefix $sagedir
+  npm install --prefix "$sagedir"
+  npm run build --prefix "$sagedir"
 
   # Remove /public from .gitignore
-  sed -i '' "s/\/public//" $sagedir/.gitignore
+  sed -i '' "s/\/public//" "$sagedir"/.gitignore
```

Comment:
does this sed work on linux? I always forget how to write a cross-compatible version.

Reply:
I believe (anecdotally) that this script has been run inside `lando ssh` which would be Linux-based so I would tentatively say yes. I also think it shouldn't be a blocker because if it doesn't work, we'll find out.